### PR TITLE
wasAssociatedWith and wasGeneratedBy replaced by associatedWith and generatedWith

### PR DESCRIPTION
**Pull request by thomasbtnfr**
According to the [issue 12](https://github.com/Inria-Empenn/BEP028_BIDSprov/issues/12), **the keys wasGeneratedBy and wasAssociatedWith have been replaced by associatedWith and generatedBy respectively**. The context.json file has been modified to make this possible.

### CODE REVIEW

File: context.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
           "@id": "prov:endedAtTime",
           "@type": "xsd:dateTime"
         },
-        "wasGeneratedBy" : {
+        "generatedBy" : {
           "@id": "prov:wasGeneratedBy",
           "@type": "@id"
         },
         "wasAttributedTo" : {
```
Comment (cmaumet) [inline context.json:28]: Could we use the same approach for all prov terms, e.g. here `wasAttributedTo` --> `attributedTo` ?

**Comment by cmaumet** [comment]
Thanks @thomasbtnfr! Looks great! I only have a tiny comment (see above) and then this is good to merge.

File: context.json (hunk 1 above)
Comment (thomasbtnfr) [reply context.json]: Sure, I just made the same changes for `attributedTo` and `informedBy`. However, I wonder what is their interest? I can't find them in the bids-prov specification.

**Review by cmaumet — APPROVED** [review]
Thanks for this PR! I have reviewed the changes, this is ready to go. Please note that as soon as this PR is merged, we will need to update `spm_load_config` so that the CONTEXT_URL points to master. @thomasbtnfr: can you take care of this? Thanks!

**Comment by thomasbtnfr** [comment]
The problem with `spm_load_config` is fixed in PR #87.

**Pull request merged**